mkosi/installer/apt.py

```diff
@@ -40,6 +40,15 @@ def __str__(self) -> str:
 
 
 class Apt(PackageManager):
+    DOCUMENTATION_GLOBS = [
```

Comment:
PEP8 only advises uppercase for global variables. For class variables lower case is fine and I think it would nicer to keep uppercase for global things only.

But maybe it would also make sense to add this as a method so that all package managers can implement this.

Reply:
We don't really need this for any other package managers since for those we don't have to extract tars